api/src/main/resources/workbench.yaml

```diff
@@ -141,38 +141,61 @@ paths:
         "204":
           description: Account deleted successfully.
 
-  /api/v1/register:
+  /api/v1/id-verification:
     post:
       tags:
         - profile
-      description: Registers a profile for the user in AllOfUs and Firecloud; called after the user has consented to the terms of use
-      operationId: register
+      description: Accepts identity information for verification.
+      operationId: submitIdVerification
       parameters:
         - in: body
-          name: registrationRequest
-          description: registration request
+          name: idVerificationRequest
           schema:
-            $ref: "#/definitions/RegistrationRequest"
+            $ref: "#/definitions/IdVerificationRequest"
       responses:
         "200":
           description: The user's profile.
           schema:
             $ref: "#/definitions/Profile"
 
-  /api/v1/id-verification:
+  # TODO: add signature / other state?
+  /api/v1/account/accept-terms-of-service:
     post:
       tags:
         - profile
-      description: Accepts identity information for verification.
-      operationId: submitIdVerification
-      parameters:
-        - in: body
-          name: idVerificationRequest
+      description: Submits consent to the terms of service for researchers.
+      operationId: submitTermsOfService
+      responses:
+        "200":
+          description: The user's profile.
           schema:
-            $ref: "#/definitions/IdVerificationRequest"
+            $ref: "#/definitions/Profile"
+
+  # TODO: add other state pertaining to ethics training?
+  /api/v1/account/complete-ethics-training:
+    post:
+      tags:
+        - profile
+      description: Submits ethics training state.
+      operationId: submitEthicsTraining
       responses:
-        "204":
-          description: Request received and validated.
+        "200":
+          description: The user's profile.
+          schema:
+            $ref: "#/definitions/Profile"
+
+  # TODO: add demographic survey response data
+  /api/v1/account/submit-demographic-survey:
```

Comment:
I have a naming suggestion. The verbs accept, complete, and submit imply some work, but I don't see that in the code itself. For clarity, based on reading the code these endpoints call, I would name them, "mark-ethics-training-completed", "mark-terms-of-service-accepted", etc.

Reply:
The demographics survey will contain answers to questions about the researcher's demographics (race, gender, etc...) this method currently is just a placeholder. So "submit" makes sense here, for ID verification, too.

Foraccepting terms of service, at a minimum I expect the user to check a box saying "I accept" or hit an "Accept" button. "Submit" still sounds appropriate there.

For ethics training, I'm not sure -- that might just be watching a video without any explicit submit action on the part of the user -- but we'll see. 

Personally, I'm OK with "submit" for all of this.

Reply:
Okay, I respect your judgement.

Reply:
Changed to "completeEthicsTraining".